backend/fief/services/email/__init__.py

```diff
@@ -4,16 +4,19 @@
 from fief.services.email.base import EmailError, EmailProvider, SendEmailError
 from fief.services.email.null import Null
 from fief.services.email.postmark import Postmark
+from fief.services.email.generic import Generic
 
 
 class AvailableEmailProvider(str, Enum):
     NULL = "NULL"
     POSTMARK = "POSTMARK"
+    GENERIC = "GENERIC"
```

Comment:
I found the settings side of things a tad confusing, but I'm hoping that these changes are enough to get things working?